src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -396,12 +397,20 @@ export class ResultsViewPageStore {
                     this.hotspots ||
                     this.cbioportalCount ||
                     this.cosmicCount ||
-                    this.driverFilter ||
+                    this.customBinaryDriverFilter ||
                     this.driverTiers.entries().reduce((oneSelected:boolean, nextEntry:[string, boolean])=>{
                         return oneSelected || nextEntry[1];
                     }, false);
 
                 return anySelected;
+            },
+
+            get customBinaryDriverFilter() {
+                return this._customBinaryDriverFilter === undefined ? AppConfig.serverConfig.oncoprint_custom_driver_annotation_binary_default : this._customBinaryDriverFilter;
+            },
+
+            get customTiersDriverFilter() {
```

Comment:
Reading the name you've chosen, I would think that it has the same sort of function as `customBinaryDriverFilter`, when in fact it's completely different - `customBinaryDriverFilter` is whether binary driver annotation is enabled, whereas this here is just the default value of tiers selected. Please try to write the names of functions and variables to reflect the functionality of how they are used. This is critically important when working on a large codebase like cbioportal, or else little by little over time the code will become unreadable. Can you please rename this to reflect better how it is used?

Reply:
@adamabeshouse I have re-evaluated the names of the two variables that I introduce and I have changed their name. I think now they reflect what they contain.